test/CrateTest/PDO/PDOTest.php

```diff
@@ -98,74 +97,20 @@ public function testInstantiationWithTraversableOptions()
     /**
      * @covers ::__construct
      */
-    public function testInstantiationWithInvalidOptions()
+    public function testInstantiationWithHttpAuth()
     {
-        $this->setExpectedException('Crate\Stdlib\Exception\InvalidArgumentException');
-
-        new PDO('crate:localhost:1234', null, null, 'a invalid value');
-    }
-
-    /**
-     * @covers ::__construct
-     */
-    public function testInstantiationWithHttpAuth() {
-        $user = 'crate';
+        $user   = 'crate';
         $passwd = 'secret';
-        $pdo = new PDO('crate:localhost:44200', $user, $passwd, []);
+        $pdo    = new PDO('crate:localhost:44200', $user, $passwd, []);
         $this->assertEquals([$user, $passwd], $pdo->getAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH));
     }
 
-    /**
-     * @covers ::setAttribute
-     */
-    public function testSetAttributeWithHttpBasicAuth()
-    {
-        $user = 'crate';
-        $passwd = 'secret';
-        $expectedCredentials = [$user, $passwd];
-
-        $client = $this->getMock(ClientInterface::class);
-        $pdo = new PDO('crate:localhost:44200', null, null, []);
-        $reflection = new ReflectionClass(PDO::class);
-        $property = $reflection->getProperty('client');
-        $property->setAccessible(true);
-        $property->setValue($pdo, $client);
-        $client
-            ->expects($this->once())
-            ->method('setHttpBasicAuth')
-            ->with($user, $passwd);
-
-        $pdo->setAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH, [$user, $passwd]);
-        $this->assertEquals($expectedCredentials, $pdo->getAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH));
-    }
-
-    /**
-     * @covers ::setAttribute
-     */
-    public function testSetAttributeWithDefaultSchema()
```

Comment:
Why did you remove this test?

Reply:
It's tested in the ServerPoolTest but it make more sense just having it in the PDOTest as well so i added it back again with c1071a7